benchmarks/benchmarks/spatial.py

```diff
@@ -425,3 +430,27 @@ def time_geometric_slerp_3d(self, num_points):
         geometric_slerp(start=self.start,
                         end=self.end,
                         t=self.t)
+
+class RotationBench(Benchmark):
+    params = [1, 10, 100]
```

Comment:
This is very much on the low side. I suggest `[10, 1000, 10000]` as in the `GeometricSlerpBench` benchmark.

Reply:
In larger scale, this pull request actually doesn't improve a lot. Converting to Cython makes these operation more efficient when called multiple times, rather than called several times with larger scale of rotaitons.

Maybe add loops in the function?

Reply:
> In larger scale, this pull request actually doesn't improve a lot

The higher numbers will quantify this though, which is useful for understanding the effects of this PR. It will also provide a baseline for future PR's.

Reply:
Got it, fixed